src/scytale.rs

```diff
@@ -0,0 +1,222 @@
+//! The Scytale cither is said to be used by ancient greeks in general, and Spartans in particular,
+//! where a leather or parchment strip is wrapped around a cylinder, called the scytale.
+//! After wrapping the strip around the scytale, the message is written horizontally, leaving the
+//! message encrypted for anyone reading without the scytale.
+//! 
+//! Because the scytale encryption is only keyed by the number of letters that fit on each roll
+//! around the scytale, meaning it is trivially cracked.
+//! 
+use common::cipher::Cipher;
+
+/// A Scytale cipher.
+///
+/// This struct is created by the `new()` method. See its documentation for more.
+pub struct Scytale {
+    height: usize,
+}
+
+impl Cipher for Scytale {
+    type Key = usize;
+    type Algorithm = Scytale;
+
+    /// Initialize a Scytale cipher with a specified key
+    /// which is the height of the columns.
+    /// Note: Will also encrypt spacing
+    ///
+    /// Returns `Err` if the `key == 0`.
+    fn new(key: usize) -> Result<Scytale, &'static str> {
+        if key == 0 {
+            Err("Invalid key, columns cannot be zero characters high")
+        } else {
+            Ok(Scytale { height: key })
+        }
+    }
+
+    /// Encrypt a message with a Scytale cipher.
```

Comment:
With this cipher, I would suggest that the message cannot contain spaces so that a 'space' may be used as a padding character (that can be stripped later during the decryption process).

Reply:
I am already doing this in some of the ciphers (asserting that spaces are not included).